jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/test/util/AbstractKieServicesBaseTest.java

```diff
@@ -110,4 +115,33 @@ protected KieFileSystem createKieFileSystemWithKProject(KieServices ks) {
         return kfs;
     }
 
+    protected void assertTask(UserTaskInstanceDesc userTask, long processInstanceId) {
+        assertTask(userTask, Long.toString(processInstanceId));
+    }
+
+    protected void assertTask(UserTaskInstanceDesc userTask, String correlationKey) {
```

Comment:
Maybe make the name more descriptive - e.g. what it's asserting on.

Reply:
Done